apps/src/templates/rubrics/RubricFloatingActionButton.jsx

```diff
@@ -71,21 +79,36 @@ function RubricFloatingActionButton({
 
   const fabIcon = aiEnabled ? aiFabIcon : rubricFabIcon;
 
+  const showPulse = isFirstSession && isFabImageLoaded && isTaImageLoaded;
+  const classes = showPulse
+    ? classNames(style.floatingActionButton, style.pulse, 'unittest-fab-pulse')
+    : style.floatingActionButton;
+
   return (
     <div id="fab-contained">
       <button
         id="ui-floatingActionButton"
-        className={style.floatingActionButton}
-        // I couldn't get an image url to work in the SCSS module, so using an inline style for now
-        style={{backgroundImage: `url(${fabIcon})`}}
+        className={classes}
         onClick={handleClick}
         type="button"
-      />
+      >
+        <img
+          alt="AI bot"
+          src={fabIcon}
+          onLoad={() => !isFabImageLoaded && setIsFabImageLoaded(true)}
+          style={{opacity: 1.0}}
```

Comment:
Maybe I'm missing a weird interaction with the animation. This could be in the stylesheet, no? Or, at least, why is it need to be set?

```suggestion
```

Reply:
good catch, moved to the stylesheet! it needs to be set in order to override this rule: https://github.com/code-dot-org/code-dot-org/blob/ec0a63a5085bae65f40a59e9ec40f737c2cf47ce/dashboard/app/assets/stylesheets/application.scss#L3330-L3331